package.json

```diff
@@ -1,25 +1,24 @@
 {
   "name": "nomidot",
-  "version": "1.0.0",
-  "main": "index.js",
-  "author": "",
+  "author": "Parity Technologies <admin@parity.io>",
+  "description": "Staking Portal for Polkadot",
   "license": "Apache-2.0",
+  "private": true,
+  "repository":"https://github.com/yjkimjunior/Nomidot",
+  "scripts": {
+    "prelerna:publish": "lerna exec yarn build --stream --scope @substrate/context --scope @substrate/ui-components",
+    "lerna:publish": "lerna version --conventional-commits && lerna publish from-git",
```

Comment:
So after this PR, we would just run: `yarn lerna:publish`. It will:
- build all the packages
- do lerna version && lerna publish

it'll still be interactive, so we can cancel if we see something's wrong.